patch/llpcPatch.cpp

```diff
@@ -166,6 +149,11 @@ void Patch::AddPasses(
     passMgr.add(createAlwaysInlinerLegacyPass());
     passMgr.add(CreatePassDeadFuncRemove());
 
+    // Check shader cache
+    auto pCheckShaderCachePass = CreatePatchCheckShaderCache();
+    passMgr.add(pCheckShaderCachePass);
+    pCheckShaderCachePass->SetCallbackFunction(checkShaderCacheFunc);
```

Comment:
How to skip the pass if the cache is hit?

Reply:
You mean if both shaders get a cache hit? In that case, both shaders get removed from the IR module, and the empty module proceeds through the passes, generating an ELF containing no code. That is then discarded. There is no way to abort a pass flow early other than by throwing and catching an error.